vmcnet/updates/optimizers.py

```diff
@@ -0,0 +1,49 @@
+# >>> from optax library
+
+from typing import Any, Optional
+
+from optax._src import base
+from optax._src import combine
+from optax._src import transform
+from optax._src.alias import ScalarOrSchedule, _scale_by_learning_rate
+
+
+def sgd(
```

Comment:
Why are we including this code now instead of just using optax?

Reply:
fixed

Reply:
removed